Install/install.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash -e
+
+GIT_REPO=https://github.com/TomWhitwell/SlowMovie
+GIT_BRANCH=master
+PYTHON_VERSION=3
+SKIP_DEPS=false
+
+function install_linux_packages(){
+  sudo apt-get update
```

Comment:
Should the sudos be in here, or should the script itself be run with sudo?

Reply:
I had an internal debate about that as I was doing it too. At least giving the user a heads up that we're using elevated commands is probably best practice. However, on the Rpi - the only platform we're targeting; the `pi` user gets sudo access without even a password by default anyway. For a hobbyist project we're trying to make easy how important is that - idk? I've written scripts that do other stuff where if you're not running as sudo it kicks out the script and asks you to re-run with privileges. That could be an option. 

I tried to find some examples - and this is by no means an exhaustive list - but two just put sudo right in the script and the other specifically says "run as sudo" in the instructions. These projects are all designed to work on an Rpi so figured they were good examples. What are you most comfortable with? I think we an justify either method. 

https://github.com/home-assistant/supervised-installer/blob/master/installer.sh
https://get.pimoroni.com/inky
https://github.com/home-assistant/supervised-installer/blob/master/installer.sh

Reply:
If all other things are equal, I think leaving sudo out and requiring the script to be run as sudo is cleaner.

Reply:
Playing around with this a bit more removing `sudo` from the script itself will make the install instructions a little more complicated looking. We'd have to do something like `sudo su -c "bash <(curl http://path/to/install.sh)" root`. It looks a lot _scarier_. An alternative would be to have the user download it first and then run `sudo ./install.sh`. 

In testing another command that would work is `curl http://path/to/install.sh | sudo bash` but that would only work for non-interactive scripts. It looks a lot more user friendly though. I could modify this for to run without prompts but we'd have to make some compromises on steps 2 and 3 (flags perhaps?)

Reply:
Interactivity and a friendly-looking command is probably more important than removing sudos from the install script. If it keeps the command simple and interactivity, I say it's worth it to have sudos in the script, especially if you found other examples where folks do that

Reply:
That makes sense.

Reply:
The only other way I could see it working, and stay with a one line install idea, is to use more flags. Something like 

```
# just do the base install, but install service by default as well
curl http://path/to/install.sh | sudo bash
```

```
# uninstall service
sudo ./install -u
```
That would still work but I'd include the service install by default and remove the menu. I don't think adding more instructions to re-run just for that will be the best user engagement either. A separate flag to uninstall if someone wanted that would be available. I think either way is doable but we will need to make some "onboarding vs usability" trade offs.